README.md

```diff
@@ -1,53 +1,56 @@
 Easy Dotfiles
 =============
 
-The `easy_setup` script was created as a solution to [Dotbot's 
-issue #22][origin-issue]. The problem: Everybody and their grandma has a way to
-manage an existing dotfiles repository. However, there isn't an existing 
-mechanism to easily setup a dotfiles repository to start with. Even if someone
-uses a dotfile manager, they still need to move files themselves, go through 
-the manager install and configuration process, and setup a repository so the 
-version control system can function.
-
-This script aims to do everything for you. All you need to do is run it, and 
-give it permission to do everything it does.
+**The problem:** Everybody and their grandma has a way to manage an existing
+dotfiles repository. However, there is no easy way create a dotfiles repository
+to start with. Even if someone uses a dotfile manager, they still need to move
+files themselves, go through the manager install and configuration process, and
+setup a repository under version control.
 
-Currently, the script is designed for use with the [DotBot dotfiles
-manager][dotbot-repo] and git.
+We aim to do all of this for you. All you need to do is run it, and
+give it permission to do everything it does.
 
-Command Line Options
---------------------
+Currently, the script is designed for use with [Dotbot][dotbot-repo] dotfile manager and git.
 
-Every command line option can have ``no-`` prepended to turn off the option.
-You can pass options in any order, but the last time an option is passed will 
-be the final state of that flag.
+Get Running in 5 Minutes
+------------------------
 
-Note that the script uses ``stderr`` whenever possible, only sending the DotBot 
-configuration file to ``stdout``. The reasons were not well thought out, but 
-hey, it's a feature.
+```bash
+  wget https://raw.githubusercontent.com/Aviator45003/easy-dotfiles/master/easy_setup.sh
```

Comment:
I think `curl` is installed by default on OS X, but not `wget`. Could we instead `curl -fsSL` it?

Also, would it be worth rewriting this to `curl ... | sh`? One less line to copy and paste.